src/tools/log-parser/Makefile

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+TARGET = kata-log-parser
+SOURCES = $(shell find . 2>&1 | grep -E '.*\.go$$')
+
+VERSION := ${shell cat ./VERSION}
+COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
+COMMIT := $(if $(shell git status --porcelain --untracked-files=no),"${COMMIT_NO}-dirty","${COMMIT_NO}")
+
+BINDIR := $(GOPATH)/bin
```

Comment:
Also IMO it should should be installed on same `BINDIR` as the runtime.

Reply:
It seems the other tools installs at the default cargo bin location, i think it's better to follow them, or to change all..

Reply:
The issue https://github.com/kata-containers/kata-containers/issues/4227 also discuss the installation of the tools, so let's handle that in a follow up PR.